docker/run_local_docker.sh

```diff
@@ -19,12 +19,12 @@ echo "Starting Local Docker Containers"
 
 docker compose down && docker compose up --build -d
 
-until curl -s -f -o /dev/null "http://localhost:9200/_cat/indices/team_search_index"; do
-  printf '.'
+until curl -s -f "http://localhost:9200/_cat/indices/team_search_index"; do
+  printf 'Checking if ES is up...\n'
```

Comment:
Nit:
```suggestion
  printf 'Checking if Elastic Search instance is up...\n'
```

Reply:
all makes sense, thanks. Will update that